packages/daimo-api/test/pushNotifier.test.ts

```diff
@@ -206,7 +206,7 @@ test("PushNotifier", async () => {
 
     assert.strictEqual(output.length, 1);
     assert.deepStrictEqual(output[0].to, ["pushTokenAlice"]);
-    assert.strictEqual(output[0].title, "$4.20 claimed");
+    assert.strictEqual(output[0].title, "Recollected $4.20");
```

Comment:
maybe "Reclaimed $4.20"? this is after you cancel your own link

Reply:
done